src/Servers/Kestrel/test/FunctionalTests/Http2/ShutdownTests.cs

```diff
@@ -100,7 +100,7 @@ public async Task GracefulShutdownWaitsForRequestsToFinish()
         }
 
         [ConditionalFact]
-        [QuarantinedTest] // Test still quarantined due to Sockets.Functional tests.
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/21521")] // Test still quarantined due to Sockets.Functional tests.
```

Comment:
Still quarantined.

Reply:
Oh, my bad - I somehow misread that as the line is being removed.